modules/installation-special-config-encrypt-disk.adoc

```diff
@@ -0,0 +1,155 @@
+// Module included in the following assemblies:
+//
+// * installing/install_config/installing-customizing.adoc
+
+[id="installation-encrypting-disks_{context}"]
+= Encrypting disks during installation
+
+During {product-title} installation, you can enable disk encryption on all master and worker nodes.
+This feature:
+
+* Is available for IPI and bare metal {product-title} deployments
+* Is supported on Red Hat CoreOS (RHCOS) systems only
+* Sets up disk encryption during the manifest installation phase so all data written to disk, from first boot forward, is encrypted
+* Encrypts data on the root filesystem only (/dev/mapper/coreos-luks-root on /)
+* Requires no user intervention for providing passphrases
+* Uses AES-256-CBC encryption
+* Should be enabled for your cluster to be FIPS compliant.
+See link:https://docs.openshift.com/container-platform/4.3/installing/installing-fips.html[FIPS compliant {product-title} clusters]
+for details.
+
+There are two different supported encryption modes:
+
+* TPM v2: This is the preferred mode. TPM v2 stores passphrases in nvram.
+To implement TPM v2 disk encryption, create an Ignition config file as described below.
+
+* Tang: To use link:https://github.com/latchset/tang[Tang] to encrypt your cluster, you need to use a Tang server.
```

Comment:
We avoid linking to GitHub repos: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites
I'm not sure of a good workaround here.

s/need to/must

Reply:
I'm going to leave this for now. I had a long talk on this subject with Eric today. I think I can convince him that this one is okay.

Reply:
Because there is not procedure for the moment without the three github sites in this document, Eric allowed them. I just had to add some disclaimers.